ocaml/idl/datamodel_host.ml

```diff
@@ -1761,6 +1761,24 @@ let apply_recommended_guidances =
       ]
     ~allowed_roles:_R_POOL_OP ()
 
+let up_to_date_state =
+  Enum
+    ( "up_to_date_state"
+    , [
+        ( "yes"
```

Comment:
I would a name that is more informative "up to date" can mean that it has the latest package metadata, or that it has installed it.
```
    ( "latest_known_packages_installed"
    , [
        ( "true"
```

Reply:
As suggested by Ming, renamed it to "latest_synced_updates_applied", as besides rpm packages, there are also livepatches to be applied.